components/navigation/navItem.module.css

```diff
@@ -6,13 +6,15 @@
   @apply flex items-center p-2 mr-4 xl:mr-4 mb-0 rounded-md;
 }
 
-.CondensedHeadingContainer {
-  @apply lg:flex-wrap lg:justify-center lg:text-center lg:w-min;
-}
+/*CondensedHedaing disabled until additional styling between md and xl is added.*/
 
-.CondensedHeadingIconContainer {
-  @apply lg:mr-0;
-}
+/* .CondensedHeadingContainer {
+  @apply xl:flex-wrap xl:justify-center xl:text-center xl:w-min;
+} */
+
+/* .CondensedHeadingIconContainer {
+  @apply xl:mr-0;
+} */
```

Comment:
@sfc-gh-tteixeira This part needs some more work. In the band between a 1024px and 1250px, the left navigation is in a collapsed state that requires hover to expand. Taking this out prevents it from looking badly aligned at least, but it's not great on a tablet since I have to press and hold to expand the left navigation.

This could (for now) just fully collapse the left navigation below 12050px and go to the mobile view. When there's more time, it might make sense for this condensed version to be fleshed out leaving a thin column to accommodate the category icons, but have the menu expand as an overlay.